charts/values.global.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+commonLabels: false
+namespace:
+  public: gmp-public
+  system: gmp-system
+version: 0.12.0
```

Comment:
Why is "version" at the top level of the new global values file? Are we saying we're going to synchronize this version across the operator, datasource-syncer and rule-evaluator?

Reply:
This comment is connected to another statement, so let's discuss both here

> Since the versions are not included in the default values file now, I would consider making them [required](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function) to make it noisier if the version is not supplied.

We used to have `appVersion` in the chart yaml for each operator/pe and rule-eval. For this purpose we had `make updateversion` to update appVersion for  operator `@$(YQ) -i '.appVersion = strenv(LABEL_API_VERSION)' ./charts/operator/Chart.yaml`. I think we meant to add same for rule-eval but forgot (?). This field was in rule-eval too:

![image](https://github.com/GoogleCloudPlatform/prometheus-engine/assets/6950331/8845199f-344a-4d3d-87c5-d8e2977688e5)

I think it only changes labels like `app.kubernetes.io/version` (with and without common), plus part of common label `app.kubernetes.io/instance`.

>  Are we saying we're going to synchronize this version across the operator, datasource-syncer and rule-evaluator?

Given what I saw I assumed we wanted that. We didn't want to synchronize those `app.kubernetes.io/version` for all manifests?

I assumed we want to template this (given sedding), thus moved this to version field in values. Datasource-syncer might need to add that label, but didn't want to change output in refactor PR yet.

Reply:
Discussed offline, sounds like we can iterate and we can keep what I have in this PR for now.